.circleci/config.yml

```diff
@@ -12,6 +13,44 @@ jobs:
           root: ~/project
           paths:
             - bin/
+  test_acc:
+    machine:
+      image: ubuntu-2004:202010-01
```

Comment:
Shouldn't it be possible to do it the same way but with the official circleci go image rather than having an orb and an ubuntu image ? 

https://circleci.com/docs/2.0/circleci-images/#next-gen-circleci-images

Reply:
I've experienced some heavy performance issues with docker runtime, that's why I moved to a real VM (machine executor).

The drawback it's that we have ton install golang manually (using an official orb)

But actually I can't manage to make acc test pass using docker runtime ...

Reply:
Ok understood !